### PR TITLE
Uploading RT ENV for Open-Vehicle-Monitoring-System-3 ,SmartEVSE-3 ,busmaster & open_evse

### PR DESCRIPTION
**Pull request by Aditi8007**
#Open-Vehicle-Monitoring-System-3 
#SmartEVSE-3 
#busmaster 
#open_evse

### CODE REVIEW

File: environment-metadata.json [hunk 1]
```diff
@@ -1043,6 +1043,186 @@
...
+        {
+            "name": "SmartEVSE-3-RT-env",
+            "version": {
+                "tag": "0.0.1",
+                "release_date": ""
+            },
+            "author": {
+                "name": "Be-Secure",
+                "type": "Organization"
+            },
```
Comment (pramit-d) [inline environment-metadata.json:1056]: @Aditi8007 , please change author name and type to BeSLab and Lab.

```
"author": {
    "name": "BeSLab",
    "type": "Lab"
}

File: environment-metadata.json [hunk 2]
```diff
@@ -1043,6 +1043,186 @@
...
+        {
+            "name": "Open-Vehicle-Monitoring-System-3-RT-env",
+            "version": {
+                "tag": "0.0.1",
+                "release_date": ""
+            },
+            "author": {
+                "name": "Be-Secure",
+                "type": "Organization"
+            },
```
Comment (pramit-d) [inline environment-metadata.json:1101]: @Aditi8007 , please change author name and type to BeSLab and Lab.

```
"author": {
    "name": "BeSLab",
    "type": "Lab"
}

File: environment-metadata.json [hunk 3]
```diff
@@ -1043,6 +1043,186 @@
...
+        {
+            "name": "open_evse-RT-env",
+            "version": {
+                "tag": "0.0.1",
+                "release_date": ""
+            },
+            "author": {
+                "name": "Be-Secure",
+                "type": "Organization"
+            },
```
Comment (pramit-d) [inline environment-metadata.json:1146]: @Aditi8007 , please change author name and type to BeSLab and Lab.

```
"author": {
    "name": "BeSLab",
    "type": "Lab"
}

File: environment-metadata.json [hunk 4]
```diff
@@ -1043,6 +1043,186 @@
...
+        {
+            "name": "busmaster-RT-env",
+            "version": {
+                "tag": "0.0.1",
+                "release_date": ""
+            },
+            "author": {
+                "name": "Be-Secure",
+                "type": "Organization"
+            },
```
Comment (pramit-d) [inline environment-metadata.json:1191]: @Aditi8007 , please change author name and type to BeSLab and Lab.

```
"author": {
    "name": "BeSLab",
    "type": "Lab"
}

**Review by pramit-d — CHANGES_REQUESTED** [review]
@Aditi8007 , **`TASK [ansible-role-oah-sonarQube: check if Docker is installed]`** is failing in all of the env scripts. Please check the issue.

File: environment-metadata.json (hunk 4 above)
Comment (Aditi8007) [reply environment-metadata.json:1191]: @pramit-d 
TASK [ansible-role-oah-sonarQube: check if Docker is installed] is working for in all of the env scripts.
unable attach Screenshot here so I sent on teams please check

**Pull request merged**